publisher/sonatype/src/main/java/eu/maveniverse/maven/njord/publisher/sonatype/SonatypeCentralPortalPublisher.java

```diff
@@ -170,9 +171,23 @@ protected void doPublish(ArtifactStore artifactStore) throws IOException {
                                 logger.info("Publishing of deployment {} succeeded: {}", deploymentId, deploymentState);
                             }
                         } catch (InterruptedException e) {
+                            Thread.currentThread().interrupt();
                             throw new IOException(e.getMessage(), e);
                         }
                     }
+                    if (publisherConfig
+                            .publishingType()
+                            .filter("AUTOMATIC"::equals)
+                            .isPresent()) {
+                        logger.info(
+                                "Publishing type is AUTOMATIC; deployment {} will be automatically published to Maven Central without further manual action (given that is is valid)",
+                                deploymentId);
+                    } else {
+                        logger.warn(
```

Comment:
I'd make this info; after all "everything went well" (as user configured) no need to raise blood pressure with warning